### DISCUSSION TITLE
[FEATURE REQUEST] Ability to favorite a prompt

### DISCUSSION THREAD

**Discussion by EliEron** [hlky/stable-diffusion-webui#601] [Feature suggestions]
If you find a prompt that produces really interesting images you currently have to store it externally in some way (I save them in a notepad document currently) which is somewhat inconvenient.

Having a button that saved the prompt along with the CFG, sampler and steps used would be quite useful. The prompt could then be displayed in a favorites list that allowed you to restore the prompt and other values with a simple press of a button.

I'm aware that there is already a proposal for a prompt history, which is somewhat similar to this proposal but not quite the same. As I would not want all prompts saved, only ones that is deemed to be particularly interesting. I would also not necessarily want seeds stored in this favorites list. As the point isn't to recreate a specific image, but to play around with a prompt that seemed to generate lots of interesting images.

**Comment by Muskelmagier** [comment]
There is already that with each newly produced picture a .yaml file is created with the same name which holds the Prompt, seed and all other parameters.

**Reply by EliEron** [reply]
Yes, I'm aware those files exists, and they are quite useful. But as mentioned my request is about the ability to mark a specific prompt as noteworthy, and to save it for easy retrieval via the GUI itself. YAML files are generated for every single file and prompt, and thus does not serve that purpose.

I currently have over a hundred prompt folders on my machine, and the number is only growing over time. Some of them are far more interesting than others, which is why I'm not asking for a general prompt history. But for a way to retrieve only prompts I have marked as interesting, which will be a far smaller curated list.
